### PR TITLE
Draft abstract for fpm GSoC project

### PR DESCRIPTION
**Pull request by LKedward**
Edits and suggestions are welcome. I will keep this in draft status until I've heard back from Jakub @kubajj about presenting this.

### CODE REVIEW

**Comment by LKedward** [comment]
Thanks for agreeing to participate Jakub @kubajj - I've put together a rough skeleton for an abstract. Please look through it and make some edits as you see fit. I believe the limit is 200 words. We'd like to get it done by the end of this week.

File: FortranCon2021-gsoc-fpm-flags/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+__Session:__ Fortran-lang minisymposium (10 min)
+
+__Abstract:__
+The Fortran Package Manager (fpm) aims to be a Fortran-specific build system to simplify compiling Fortran code and using third-party dependencies.
+Fpm is currently in alpha development status and supports a wide range of features.
+Recent work, undertaken as part of the Google Summer of Code programme, has implemented the syntax and logic required to provide fine-grain control of compiler flags by package maintainers.
+Specifying compiler flags for fpm projects is a complex problem due the existence of multiple Fortran compilers, each with their own set of flags.
+Moveover, package maintainers often need to specify different flags for a variety of build profiles, such as 'debug' and 'release', as well as across a variety of target operating systems.
+
+In this presentation, detail is given on the new functionality for specifying compiler flags in fpm including: the syntax for the manifest file; the behaviour for package dependencies; and common use-case examples.
```
Comment (everythingfunctional) [inline FortranCon2021-gsoc-fpm-flags/README.md:17]: I would also include describing the nature of the problem we're trying to solve; that some packages don't necessarily work well with the default compiler flags, or a particular set is desired for testing or experimentation, and that the current practice of specifying them manually via the command line is not convenient.

File: FortranCon2021-gsoc-fpm-flags/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+__Co-Authors:__ [Laurence Kedward](https://github.com/LKedward), [Brad Richardson](https://github.com/everythingfunctional), [Sebastian Ehlert](https://github.com/awvwgk)
+
+__Session:__ Fortran-lang minisymposium (10 min)
+
+__Abstract:__
+The Fortran Package Manager (fpm) aims to be a Fortran-specific build system to simplify compiling Fortran code and using third-party dependencies.
+Fpm is currently in alpha development status and supports a wide range of features.
+Recent work, undertaken as part of the Google Summer of Code programme, has implemented the syntax and logic required to provide fine-grain control of compiler flags by package maintainers.
+Specifying compiler flags for fpm projects is a complex problem due the existence of multiple Fortran compilers, each with their own set of flags.
+Moveover, package maintainers often need to specify different flags for a variety of build profiles, such as 'debug' and 'release', as well as across a variety of target operating systems.
```
Comment (kubajj) [inline FortranCon2021-gsoc-fpm-flags/README.md]: I like the text as is quite a lot, but do you think that something similar to what follows could be added after this sentence? @everythingfunctional
```What is more, some packages require specific compiler flags other than the default ones built into the fpm and complex projects with various package dependencies might need specific flags for each package, therefore a priority system for compiler profiles was designed and implemented.```
I have not figured out how to edit the file directly yet.

**Comment by LKedward** [comment]
This abstract has now been submitted. Thanks Jakub and Brad!

**Pull request merged**